js/okex.js

```diff
@@ -1248,6 +1249,54 @@ module.exports = class okex extends Exchange {
         return this.parseOHLCVs (data, market, timeframe, since, limit);
     }
 
+    async fetchFundingRateHistory (symbol, limit = undefined, since = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'instId': market['id'],
+        };
+        if (since !== undefined) {
+            request['after'] = since;
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        const response = await this.publicGetPublicFundingRateHistory (this.extend (request, params));
+        //
+        //     {
+        //         "code":"0",
+        //         "msg":"",
+        //         "data":[
+        //             {
+        //                 "instType":"SWAP",
+        //                 "instId":"BTC-USDT-SWAP",
+        //                 "fundingRate":"0.018",
+        //                 "realizedRate":"0.017",
+        //                 "fundingTime":"1597026383085"
+        //             },
+        //             {
+        //                 "instType":"SWAP",
+        //                 "instId":"BTC-USDT-SWAP",
+        //                 "fundingRate":"0.018",
+        //                 "realizedRate":"0.017",
+        //                 "fundingTime":"1597026383085"
+        //             }
+        //         ]
+        //     }
+        //
+        const rates = [];
+        const data = this.safeValue (response, 'data');
+        for (let i = 0; i < data.length; i++) {
+            const rate = data[i];
+            rates.push ({
+                'symbol': this.safeString (rate, 'instId'),
+                'fundingRate': this.safeNumber (rate, 'realizedRate'),
```

Comment:
you need to fill out all the keys, even if they are undefined

Reply:
All the exchanges only return these 3 properties

Reply:
ok, i would have expected that we had the datetime here as well, my bad

Reply:
> ok, i would have expected that we had the datetime here as well, my bad

I can add it, the method is new

Reply:
@samgermain adding the timestamp would be great, thanks!

Reply:
I added it, when I checked the other exchanges, I saw that it had already been changed so I thought it was done, but I forgot about this one

Reply:
thx!

yeah i added it to other exchanges in #10398